### PR TITLE
add tests through GitHub actions

### PR DESCRIPTION
**Pull request by sabjorn**
tests on `push`, but this might not be ideal long term

### CODE REVIEW

**Comment by sabjorn** [comment]
@y-brehm note: python 3.11 fails if I try to run tests with it. Could be the requirements versions don't work with 3.11.

**Comment by y-brehm** [comment]
> @y-brehm note: python 3.11 fails if I try to run tests with it. Could be the requirements versions don't work with 3.11.

https://github.com/y-brehm/waveAlign/pull/22 I fixed this with my recent PR.

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
```
Comment (y-brehm) [inline .github/workflows/run_tests.yml:10]: I think you can add 3.11 here after I merged my branch.

**Review by y-brehm — APPROVED** [review]
Looks good. Maybe you wait until I merged my branch to fix python 3.11. Then you can rebase and add 3.11 to the run_tests.yml.

**Pull request merged**